Backend/app/rider-platform/rider-app/Main/spec/Storage/beckn-config.yaml

```diff
@@ -21,13 +21,14 @@ BecknConfig:
     vehicleCategory: VehicleCategory
     collectedBy: PaymentCollectedBy
     staticTermsUrl: Maybe BaseUrl
-    buyerFinderFee : Maybe Text
+    buyerFinderFee: Maybe Text
```

Comment:
Fix: this should be `HighPrecMoney`

can be fixed in next PR @RitikaHanish @VD321